pkg/provider/provider.go

```diff
@@ -98,6 +98,15 @@ type KeyVaultObject struct {
 	// The encoding of the object in KeyVault
 	// Supported encodings are Base64, Hex, Utf-8
 	ObjectEncoding string `json:"objectEncoding" yaml:"objectEncoding"`
+	// Permission is the file permissions
+	Permission string `json:"permission" yaml:"permission"`
```

Comment:
How about `filePermission` for the field name? That's more specific on what the field is intended for.

Reply:
Sure.. Thought the same but hoping to keep name consistent with driver mount request. I am ok with either name.